todo-winjs/WinJSTodo/WinJsTodo.jsproj

```diff
@@ -1,128 +1,117 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
(Note: github won't let me comment directly on the images, that's why this is here)

Please restore the various images that were overwritten below (i.e. the splashscreens and icons). They used to be checkmarks and now are defaulted to the "X in a box" picture.